cmd/xmidt-agent/wrphandlers.go

```diff
@@ -92,6 +95,33 @@ func provideAuthHandler(in authIn) (*auth.Handler, error) {
 	return h, err
 }
 
+type crudIn struct {
+	fx.In
+
+	XmidtAgentCrud  XmidtAgentCrud
+	Identity        Identity
+	Egress          websocket.Egress
+	LogLevelService *loglevel.LogLevelService
+	PubSub          *pubsub.PubSub
+}
+
+func provideCrudHandler(in crudIn) (*xmidt_agent_crud.Handler, error) {
+	h, err := xmidt_agent_crud.New(in.Egress, string(in.Identity.DeviceID), in.LogLevelService)
+	if err != nil {
+		err = errors.Join(ErrWRPHandlerConfig, err)
+		return nil, err
+	}
+
+	// what do we do with the return value?  Does this have to be passed to pubSub somehow? Seems
```

Comment:
It's safe to ignore it.  Only if you need the ability to cancel the subscription does it matter.